WildFly_28.0.0.Final/microprofile-6.0-selected-specifications/microprofile-6.0-selected-specifications-certification.adoc

```diff
@@ -0,0 +1,31 @@
+= TCK Results
+:ext-relative: {outfilesuffix}
+:imagesdir: ../../images/
+
+image:splash_wildflylogo_small.png[WildFly, align="center"]
+
+[[introduction]]
+== Introduction
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for the selected specifications from the release of https://github.com/eclipse/microprofile/releases/tag/6.0[MicroProfile 6.0]. This certification contains all specifications defined in https://github.com/eclipse/microprofile/releases/tag/6.0[MicroProfile 6.0] except for MicroProfile Metrics which is explicitly excluded as part of the Red Hat position statement.
+
+=== WildFly Full Platform, Certification Summary
+----
+Date:  Fri Apr 21 01:19:49 PM CEST 2023
+Product Name:  WildFly
+Product Version: WildFly 28.0.0.Final
+URL to product download page: https://wildfly.org/downloads
+
+Product Description: An implementation of the Jakarta EE Full Platform, Web Profile, and MicroProfile Specifications
```

Comment:
NBD but we can add "Core Profile".

s/MicroProfile/selected MicroProfile/g

s/Specifications/specifications.

I think still mentioning "MicroProfile" is okbecause we do implement MicroProfile specifications, just not MicroProfile Platform.

Reply:
yes, correct, we can mention MicroProfile